en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
 
-### Description of the group
+### Description
 
-A description of the group, to help you remember what it is about. Hovering the mouse over the group name displays this description.
+A description of the group, to help you remember what it is about.
+This description is displayed when hovering the mouse over the group name.
 
 ### Icon and color
 
-An icon can be displayed in front of the group name. Choose your favorite icon among the ones available at [https://materialdesignicons.com/](https://materialdesignicons.com/), and enter its name of the field _Icon_. The color of the icon can be set in to the field **Color**.
+An icon can be displayed in front of the group name.
+Choose your favorite icon among the ones available at [https://materialdesignicons.com/](https://materialdesignicons.com/), and enter its name of the field _Icon_.
+The color of the icon can be set in to the field _Color_.
+
+**%%% screenshot here: part of group pane showing icon and color %%%**
+
+### Hierachical context
+
+The displayed entries depend on the hierarchical context of the group.
+When a group is selected, the displayed entries can be:
+
+- **independent** of its supergroup and of its subgroups.
+- a **union** between the entries of the group and of its subgroups.
+- an **intersection** between the entries of the group and of its supergroup.
+
+#### Independent group
+
+By default, a group is **independent** of its position in the groups tree: When selected, the table of entries shows only the group's content (all of its entries).
+
+#### Intersection between a group and its supergroup
+
+For a group defined with a hierarchical context *Intersection*, only the entries contained in *both* the group and its supergroup are displayed when the group is selected.
+
+This is especially relevant for groups based on keywords or search expression, where it is often useful to define a group that intersects its supergroup.
+For example, consider a supergroup containing entries with the keyword *distribution* and a subgroup containing entries with the keyword *gauss*.
+With the subgroup *gauss* defined as an intersection (of its supergroup), selecting the subgroup *gauss* displays entries that match both conditions, i.e. are concerned with Gaussian distributions.
+Note that entries that only belong to the subgroup *gauss* will not be shown, i.e. for an entry to be displayed when selecting *gauss*, it must be assigned to both the subgroup *gauss* and the supergroup *distribution*.
+By adding another intersection group for *laplace* to the supergroup *distribution*, the grouping can easily be extended to Laplace distributions.
+
+### Union between a group and its subgroups
+
+The union of a group and its subgroups is the logical complement of the intersection: when defined as *union*, selecting the group displays *both* the group's own entries and its subgroups' entries.
+
+**%%%Help needed: Add an example of a sensible use here. %%%**
```

Comment:
Parent group: some topic (say music)
Subgroups: some subtopic (say classical muscic, Jazz, ...)

Than you can simply add articles dealing with a specific aspect to the corresponding subtopic, and they automatically appear in the topic group as well (without additional work).

Reply:
Thanks for the suggestion. Added!